tests/functional/firefox/test_sync.py

```diff
@@ -23,6 +23,13 @@ def test_android_download_button_displayed(base_url, selenium):
 
 @pytest.mark.smoke
 @pytest.mark.nondestructive
+def test_ios_download_button_displayed(base_url, selenium):
```

Comment:
Oh, certainly. Wasn't sure if it was better to have independent or combined tests. Will update.